gcp_airflow_foundations/common/utils/convert_config.py

```diff
@@ -36,11 +36,23 @@ def convert_template_to_table(template_config: SourceTemplateConfig, table_name:
         "ods_config",
         "hds_config",
         "start_date",
-        "extra_options"
     ]
 
     for of in optional_fields:
         if hasattr(template_config, of):
             params[of] = getattr(template_config, of)
 
+    template_extra_options = getattr(template_config, "extra_options")
+    table_extra_options = {}
```

Comment:
I would consider putting this block of code as a separate function ('get_extra_options' or something like that), and unit test it. Would make the code easier to understand. 
Perhaps add a couple of comments to the function to explain how it works